### PR TITLE
ABI Encoder Redesign

### PR DESCRIPTION
**Pull request by fhildeb**
This PR introduces **updated headings, notes, links, and description**

### ABI Encoder

- Added external links for every transaction type

### Screenshot
Bildschirmfoto 2023-11-20 um 13 47 08 1
Bildschirmfoto 2023-11-20 um 13 48 23

### CODE REVIEW

File: components/Encode/Encode.tsx [hunk 1]
```diff
@@ -32,6 +32,14 @@ const Encode: React.FC<Props> = ({ web3 }) => {
                 onChange={() => setMode(TRANSACTION_TYPES.SET_DATA)}
               />
               setData
+              <a
+                href="https://github.com/ERC725Alliance/ERC725/blob/main/docs/ERC-725.md#setdata"
```
Comment (Hugoo) [inline components/Encode/Encode.tsx:36]: I'd recommend this page instead

https://docs.lukso.tech/contracts/contracts/ERC725/#setdata

Wdyt?

File: components/Encode/Encode.tsx [hunk 2]
```diff
@@ -43,6 +51,14 @@ const Encode: React.FC<Props> = ({ web3 }) => {
                 onChange={() => setMode(TRANSACTION_TYPES.SET_DATA_BATCH)}
               />
               setDataBatch
+              <a
+                href="https://github.com/ERC725Alliance/ERC725/blob/main/docs/ERC-725.md#setdatabatch"
```
Comment (Hugoo) [inline components/Encode/Encode.tsx:55]: https://docs.lukso.tech/contracts/contracts/ERC725/#setdatabatch

File: components/Encode/Encode.tsx [hunk 3]
```diff
@@ -54,6 +70,14 @@ const Encode: React.FC<Props> = ({ web3 }) => {
                 onChange={() => setMode(TRANSACTION_TYPES.EXECUTE)}
               />
               execute
+              <a
+                href="https://github.com/ERC725Alliance/ERC725/blob/main/docs/ERC-725.md#execute"
```
Comment (Hugoo) [inline components/Encode/Encode.tsx:74]: Same docs link instead of raw specs in the repo

File: pages/abi-encoder.tsx [hunk 4]
```diff
@@ -25,34 +24,72 @@ const Home: NextPage = () => {
...
         <article className="message is-info">
           <div className="message-body">
-            This tool lets you encode and decode{' '}
-            <a href="https://docs.lukso.tech/standards/smart-contracts/lsp0-erc725-account">
-              LSP0 ERC725 Account
-            </a>{' '}
-            transactions.
+            This tool will encode and decode transaction data of
+            <a
+              className="mx-1"
```
Comment (Hugoo) [inline pages/abi-encoder.tsx:32]: Not sure why you need this but I'm on mobile so I can't test locally

**Review by Hugoo — COMMENTED** [review]
If you could just update the links to the docs instead the. We can merge

File: pages/abi-encoder.tsx (hunk 4 above)
Comment (fhildeb) [reply pages/abi-encoder.tsx:32]: Its either `mx-1` on every link or `{' '}` behind AND in front of the element. Class looks much nicer and failsafe imo. Also saw it in the examples of Bulma.

**Review by Hugoo — APPROVED** [review]

**Pull request merged**